AndroidTest/app/src/main/java/cloudant/com/androidtest/MyActivity.java

```diff
@@ -15,112 +15,19 @@
 package cloudant.com.androidtest;
```

Comment:
Can we update this file with a comment explaining why it exists (to satisfy the android test system and to set the test properties)? Also it's not obvious how `setTestProperties` gets called now.

Finally it might be a good idea to change the name from MyActivity to TestActivity or something more descriptive.